spec/features/reservations_spec.rb

```diff
@@ -376,4 +414,214 @@
       end
     end
   end
+
+  context 'valid items on confirmation page' do
+    before(:each) do
```

Comment:
Based on some manual testing (running `save_and_open_page` at various stages of the specs to see what was going on), the login code is being run *after* this `before` block, meaning that after login it's probably taking you to the catalog, not to the `new_reservation_path`.

I'm not sure what the best way to resolve this is; we might just need to add the `visit new_reservation_path` line to the `before` block in each later `context` so it comes after login. I'll do a little more testing.